CHANGELOG.md

```diff
@@ -4,7 +4,34 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased / In Rust Beta or Nightly
 
-[1b89724...master](https://github.com/rust-lang/rust-clippy/compare/1b89724...master)
+[1fac380...master](https://github.com/rust-lang/rust-clippy/compare/1fac380...master)
+
+## Rust 1.34 (2019-04-10)
+
+[1b89724...1fac380](https://github.com/rust-lang/rust-clippy/compare/1b89724...1fac380)
+
+* New lint: [`assertions_on_constants`] to detect for example `assert!(true)`
+* New lint: [`dbg_macro`] to detect uses of the `dbg!` macro
+* New lint: [`missing_const_for_fn`] that can suggest functions to be made `const`
+* New lint: [`too_many_lines`] to detect functions with excessive LOC
```

Comment:
Perhaps the configuration option `too-many-lines-threshold: u64` should be mentioned here as well.
What do you think?

Reply:
Yeah, makes sense to mention the configuration options for new lints :+1: